Tasks/AzurePowerShellV5/AzurePowerShell.ps1

```diff
@@ -76,6 +76,13 @@ try
         $contents += ". '$("$scriptPath".Replace("'", "''"))' $scriptArguments".Trim()
     }
 
+    $contents += 'if (!(Test-Path -LiteralPath variable:\LASTEXITCODE)) {'
```

Comment:
This issue was not occuring in powershell and refered this: https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/PowerShellV2/powershell.ps1#L60 for change

Reply:
@20shivangi this behaviour is tied to the 'ignore last exit code' input (see powershell task task.json file). The current change might cause scripts to fail.

Reply:
According to this ignore exit code input, by default its value is false due to which "if ((Test-Path -LiteralPath variable:\\LASTEXITCODE)) { exit $LASTEXITCODE }" this will be appended at the end of script.

So, when we are doing the same in AzurePowerShell task (default behavior), if the user is providing exit 0, it is passing, but in case he provides exit 1, the task fails. I am unable to get in what scenario this change will cause failure.

Reply:
Suppose today, a user is running a script that runs without throwing any PowerShell exceptions. But say the script also invokes another process (like say net.exe) and this process exits with exit code not 0. Then, even though the last exit code is non zero, the task succeeds. But now, if the lastexitcode starts being returned, won't the task fail ? Won't this be a breaking change ??